src/main/java/nl/delftelectronics/spaceinvaders/core/scenes/MenuScene.java

```diff
@@ -43,6 +50,13 @@ public void labelClicked(LabelEntity label) {
 		if (label == quitButton) {
 			System.exit(0);
 		}
+		if (label == loadGameButton) {
+			Optional<GameInformation> gi = GameInformation.load();
+			if (gi.isPresent()) {
+				Engine engine = Engine.getInstance();
+				engine.setScene(new StoreScene(scene, gi.get()));
+			}
```

Comment:
Can't you do something if the load fails? Maybe display something on the screen?